.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: check-yaml
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+  - id: flake8
+    args: ["--ignore=F405,E731,F403,W503,E722,E203", --max-line-length=120]
+    # F405 : Name may be undefined, or defined from star imports: module
+    # E731 : Do not assign a lambda expression, use a def
+    # F403 : 'from module import *' used; unable to detect undefined names
+    # W503 : Line break before binary operator
+    # E722 : Do not use bare 'except'
+    # E203 : Whitespace before ':'
+
+- repo: https://github.com/pycqa/isort
```

Comment:
I hate it...
You'll be very careful letting it messing with your imports. It may break your code

Reply:
But normally it's fine.

Reply:
Actually, isort has conflict with black becase they use different code style... but it dose can beautify the code...